src/phases/execution/clusteredCollection.yml

```diff
@@ -0,0 +1,128 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Run basic insert and find and delete workload on a collection clustered by {_id: 1} .
+  Clustered collections are planned to mainly serve operations over the cluster
+  key such as monotonically increasing inserts, range queries and range deletions.
+
+GlobalDefaults:
+  RecordIdExtraPaddingParam: &RecordIdExtraPaddingParam {^Parameter: {Name: "RecordIdExtraPadding", Default: 0}}
+
+Actors:
+
+# Phase 1: create collection
+- Name: Create
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        create: &coll Collection0
+        clusteredIndex: {key: {_id: 1}, unique: true}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: insert. TODO Make multithreaded once TIG-2938 is resolved.
+- Name: Insert
+  Type: Loader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: &docCount 999999
```

Comment:
Urgh, that is a bit ugly. Is the count changed because the ^RandomInt max is inclusive combined with the inability to do basic maths in yml.

Reply:
Exactly, and thanks for confirming this is a known limitation. @jimoleary there a TIG ticket to link this against, as I'm going to document why I can't round to 1'000'000?

Reply:
Would indexing from 1 solve the issue? That is:
```
    DocumentCount: &docCount 1000000
    BatchSize: 1000
    Document:
      _id: {^Join: {array: [{ ^FormatString: {"format": "%06d", "withArgs": [{^Inc: {start: 1}}]}}, {^FastRandomString: {length: *RecordIdExtraPaddingParam, alphabet: "a"}}]}}

....

        Filter: {_id: {^Join: {array: [{ ^FormatString: {"format": "%06d", "withArgs": [{^RandomInt: {min: 1, max: *docCount}}]}}, {^FastRandomString: {length: *RecordIdExtraPaddingParam, alphabet: "a"}}]}}}

````

I've raised [TIG-3759](https://jira.mongodb.org/browse/TIG-3759) to track it for now.